antispam-bee.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * Plugin Name: Antispam Bee
+ * Description: Antispam plugin with a sophisticated toolset for effective day to day comment and trackback spam-fighting. Built with data protection and privacy in mind.
+ * Author:      pluginkollektiv
+ * Author URI:  https://pluginkollektiv.org
+ * Plugin URI:  https://wordpress.org/plugins/antispam-bee/
+ * Text Domain: antispam-bee
+ * Domain Path: /lang
+ * License:     GPLv2 or later
+ * License URI: http://www.gnu.org/licenses/gpl-2.0.html
+ * Version:     3.0-dev
+ *
+ * @package Antispam Bee
+ **/
+
+declare(strict_types = 1);
+
+namespace Pluginkollektiv\AntispamBee;
+
+use Pluginkollektiv\AntispamBee\Config\AntispamBeeConfig;
+use Pluginkollektiv\AntispamBee\Config\GenericWPOption;
+use Pluginkollektiv\AntispamBee\Entity\CommentDataFactory;
+use Pluginkollektiv\AntispamBee\Exceptions\Runtime;
+use Pluginkollektiv\AntispamBee\Filter\FilterFactory;
+use Pluginkollektiv\AntispamBee\Filter\FilterInterface;
+use Pluginkollektiv\AntispamBee\Option\OptionFactory;
+use Pluginkollektiv\AntispamBee\Handler\CommentSpamHandler;
+use Pluginkollektiv\AntispamBee\Helper\IP;
+use Pluginkollektiv\AntispamBee\Logger\FileLogger;
+use Pluginkollektiv\AntispamBee\PostProcessor\PostProcessorFactory;
+use Pluginkollektiv\AntispamBee\PostProcessor\PostProcessorInterface;
+use Pluginkollektiv\AntispamBee\Repository\FilterRepository;
+use Pluginkollektiv\AntispamBee\Repository\PostProcessorRepository;
+use Pluginkollektiv\AntispamBee\Repository\ReasonsRepository;
+use Pluginkollektiv\AntispamBee\Settings\Controller;
+
+if ( ! class_exists( AntispamBeeConfig::class ) ) {
+	require_once __DIR__ . '/vendor/autoload.php';
+}
+
+add_action(
+	'plugins_loaded',
+	function() {
+
+		global $wpdb;
+
+		$option_factory = new OptionFactory();
```

Comment:
Mix from WP Code Style and PSR12? I mean the WP Code Style needs underline between the words.